code/game/jobs/job_exp.dm

```diff
@@ -13,12 +13,13 @@
 	var/jtext
 	for(var/client/C in clients)
 		jtext = "No Job"
-		if(C.mob.mind.assigned_role)
-			theirjob = job_master.GetJob(C.mob.mind.assigned_role)
-			if(theirjob)
-				jtext = theirjob.title
-				if(config.use_exp_restrictions && theirjob.exp_requirements && theirjob.exp_type)
-					jtext += "<span class='warning'>*</span>"
+		if(C.mob.mind)
+			if(C.mob.mind.assigned_role)
```

Comment:
`if(C.mob.mind && C.mob.mind.assigned_role)` will condense this to one line and one indent